### ISSUE TITLE
Mullvad Browser disables uBlock Origin

### ISSUE THREAD

**Issue by Badwolfe24** [mullvad/mullvad-browser#293]
Mullvad browser (fresh copy with no user-modifications) keeps disabling the uBlock Origin extension.

I posted this on Privacy Guides and had a response from "ruihildt" that directed me to uninstall and then reinstall a fresh copy of Mullvad browser because uBlock is included by default.

That worked for about 22 days.

It just happened again. Mullvad browser disabled uBlock. If you remember, you had me delete and reisntall a fresh Mullvad browser.

I have not gone through the process of deleting and reinstalling Mullvad in case you needed me to research something on the browser.

uBlock just got disabled in the fresh Mullvad browser. I took screen shots to show. Why would this happen?
![uBlock disabled again](https://github.com/user-attachments/assets/2591af61-577c-4a6b-b966-5c1f14ae7f79)
![mullvad browser version](https://github.com/user-attachments/assets/80c6fce3-7688-4607-88da-4f9e40546d47)

**Comment by Thorin-Oakenpants** [comment]
@hackademix FYI

**Comment by hackademix** [comment]
It looks lile a bad interaction with (or a bug within) AMO's cryptographic signing verification.

@Badwolfe24:
 
1. which uBlock version is installed and disabled? 
2. Where that [More Information] button leads you to?

**Comment by Badwolfe24** [comment]
> It looks lile a bad interaction with (or a bug within) AMO's cryptographic signing verification.
> @Badwolfe24:
>     1. which uBlock version is installed and disabled?
>     2. Where that [More Information] button leads you to?
-----------------------
uBlock Origin 1.59.0

I'm new to Github and wasn't sure I was going to even get a response so I deleted Browser and reinstalled a fresh copy (unlike my original plan of suffering without uBlock).  The ads were just too much to deal with, without uBlock's help, so I don't have that [More information] button at this time.
When it happens again, I will grab that info.

**Comment by ruihildt** [comment]
@Badwolfe24 Indeed that was me on the forum as well.
I can't help you more, but I see that @hackademix has been trying to help you.

**Comment by ruihildt** [comment]
@Badwolfe24 Do you still have the same issue?

**Comment by Badwolfe24** [comment]
Yes, it just happened again last week. I corresponded with Stefan of Mullvad support about it.

Symptoms: uBlock will disappear on a clean, default only Mullvad Browser.

- I find uBlock disabled. I took a lot of screenshots to show what I found.
- Then, a week later, it becomes enabled again.

According to Stefan, Mullvad will "poll" uBlock extension to make sure its signature is a valid extension. If it finds that Mullvad cannot authenticate it, it will disable the extension.
We are guessing that the a future "polling" finds the signature valid and re-enables it.

I am now waiting for the next time it becomes disabled to validate this.

Stefan states that I am the only person this happens to though. That I find strange for sure.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.
